arches/install/arches-templates/project_name/hosts.py-tpl

```diff
@@ -0,0 +1,7 @@
+import re
+from django_hosts import patterns, host
+
+host_patterns = patterns(
+    "",
+    host(re.sub(r"_", r"-", r"{{ project_name }}"), "{{ project_name }}.urls", name="{{ project_name }}"),
```

Comment:
yeah it looks unnecessary but it's how it's [outlined in the docs](https://django-hosts.readthedocs.io/en/latest/)

Reply:
no problem to leave this as is, but I'm not seeing `re.sub()` in the docs?

Reply:
ah yeah this is because underscores and domains do not play nicely together, and `project_name` can contain underscores.

Adding a domain with an underscore results in:

```
Invalid HTTP_HOST header: 'foo_bar.localhost:8000'. The domain name provided is not valid according to RFC 1034/1035.
```

Reply:
Yeah yeah, I just mean re.sub() returns a not-raw string anyway, so we're already providing a not-raw string -- in that case, no need to pass raw string _args_ to re.sub() such as `"arches"`, which isn't a regex. No matter.